app/assets/stylesheets/admin/login.scss

```diff
@@ -0,0 +1,7 @@
+.vcenter{
+margin-top: 140px;
+
+}
+.leftcenter{
+margin-left:550px;	
+}
```

Comment:
Indentation issues. Please use below to beautify your css with 2 spaces indentation.  

http://codebeautify.org/css-beautify-minify